extensions/scarb-doc/Cargo.toml

```diff
@@ -26,6 +26,7 @@ indoc.workspace = true
 itertools.workspace = true
 scarb-metadata = { path = "../../scarb-metadata" }
 scarb-ui = { path = "../../utils/scarb-ui" }
+scarb = { path = "../../scarb" }
```

Comment:
please don't do this, we want scarb to be a binary in the future, unusable as a crate

I'll add comments in all usage places how to do them differently

Reply:
Removed it completely as we don't need it right now (using metadata command for features)